Style.css

```diff
@@ -1,186 +1,202 @@
 @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap');
 
-:hover{
-    cursor:grab;   
-}
-:active{
-    cursor: grabbing;
-}
-body{
-    
-    margin-top: 50px;
-    margin-bottom: 50px;
-    margin-left: auto;
-    margin-right: auto;
-    
-}
-form{
-    width: 900px;
-    height: 850px;
-    margin-top: 20px;
-    margin-left: auto;
-    margin-right: auto;
-    background-image: url(https://images.unsplash.com/photo-1550312462-2e5baee16edb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1yZWxhdGVkfDE1fHx8ZW58MHx8fHw%3D&auto=format&fit=crop&w=500&q=60);
-    border: 1px solid #ccc;
-}
- .header img{
-    height: 45px;
-    width: 45px;
-    padding-right: 10px;
-    padding-left: 10px;
-    float: left;
-    display: inline;
-       
-} 
-
-#img1{
-    margin-top: 20px;
-    width: 80px;
-    margin-right: 160px;
-    margin-left: 10px;
-
-}
-.container i{
-    width: 20px;
-}
-.header h2{
-    text-align: left;
-   margin-top: 6px;
-   position: absolute;
+* {
+  font-family: 'Open Sans';
+}
```

Comment:
While Open Sans was imported, it wasn't being used. The use of * is not recommended much but I've used it for quickly setting all text as Open Sans.